addons/website_event/static/src/js/website_event_ticket_quantity_selection.js

```diff
@@ -0,0 +1,36 @@
+odoo.define('website_event.ticket_qty_selection', require => {
+'use strict';
+
+const publicWidget = require('web.public.widget');
+const core = require('web.core');
+
+/**
+ * This widget can be hooked to a select tag in order to send an event "event_ticket_quantity_change" each time the
+ * selection is changed.
+ * The event payload contains ticketId:{integer}, quantity:{integer}.
+ *
+ * @param ticketId the id of the ticket to which the selection refer to
```

Comment:
line length, grammar "en passant"
```suggestion
 * This widget can be hooked to a select tag in order to send an
 *  "event_ticket_quantity_change" event each time the selection is changed.
 * The event payload contains ticketId:{integer}, quantity:{integer}.
 *
 * @param ticketId id of the ticket to which the selection refers
```

Reply:
Updated, thanks